src/components/MyKiva/BadgeModalContentJourney.vue

```diff
@@ -257,6 +262,31 @@ const journeyDescription = computed(() => {
 
 const RightLeaningLine = defineAsyncComponent(() => import('#src/assets/images/right-leaning-line.svg'));
 const LeftLeaningLine = defineAsyncComponent(() => import('#src/assets/images/left-leaning-line.svg'));
+
+const handleToggleGradient = e => {
+	const atBottom = e.target.scrollTop + e.target.clientHeight + 1 >= e.target.scrollHeight;
+	if (toggleGradient.value !== atBottom) {
+		toggleGradient.value = atBottom;
+		emit('toggle-gradient', toggleGradient.value);
+	}
+};
+
+const throttledHandleToggleGradient = _throttle(handleToggleGradient, 100);
+
+onMounted(() => {
+	// Sidesheet scrollable section
+	scrollEl.value = document.querySelector('.tw-overflow-y-auto.tw-overscroll-y-contain');
```

Comment:
As classes could change in the future, probably adding an id is better approach.